### ISSUE TITLE
[REQUEST] More ways of Cooling

### ISSUE THREAD

**Issue by hideki2k02** [GabrielOlvH/Industrial-Revolution#531]
**Description:**
Hello, it would be nice if there was additional ways of cooling. Could be anything, but i was imagining something like IC2 Vents as they're simple: Standard, Overclocked, Advanced. 

Tech Reborn sometimes also make Multi-Block Machines behave differently based on what Machine Frame, so maybe do the same for Factory Intakes? (Basic Intake, Overclocked Intake etc)

**Reason:**
When you start to go far in automation, it is really easy to overheat your machines if you do not have a multiple-machine setup.

**Comment by antonhibl** [comment]
I agree a better cooling system than the mediocre fans(if under any stress they last an hour or so max, take the industrial smelter for example) would be nice.
